pkg/translator/translator_test.go

```diff
@@ -97,6 +97,57 @@ func TestGenerateConfig(t *testing.T) {
 	}
 }
 
+func TestGenerateRefineryRulesConfigs(t *testing.T) {
```

Comment:
not a blocker, but i would add the desired generated config type to the test struct and avoid the duplicated code (maybe include it in the test name to make it clear which config failed to generate